test/integration/inherited-entities/auth.test.js

```diff
@@ -187,4 +187,68 @@ describe('Inherited Auth', function () {
             expect(response.code).to.eql(200);
         });
     });
+
+    describe('in collection and request level', function () {
+        before(function (done) {
+            var clonedRunOptions = _.merge({}, runOptions,
```

Comment:
Remove the first argument to `_.merge`, and shift `runOptions` to after the collection.

Reply:
@kunagpal we want the options to override `runOptions` always, but not mutate it. `_.defaults` does not do a deep merge.